doc/ClassicalMaximals.bib

```diff
@@ -31,6 +31,24 @@ @article {HR05
        URL = {https://doi.org/10.1112/S1461157000000899},
 }
 
+%%%%%%%%%%%%%%%%%%%%%%%%%%%
+% TODO
+% Add MRNUMBER, MRREVIEWER
+%%%%%%%%%%%%%%%%%%%%%%%%%%%
+@article {HR10,
+    AUTHOR = {Holt, Derek F. and Roney-Dougal, Colva M.},
+     TITLE = {Constructing maximal subgroups of orthogonal groups},
+   JOURNAL = {LMS J. Comput. Math.},
+  FJOURNAL = {LMS Journal of Computation and Mathematics},
+    VOLUME = {13},
+      YEAR = {2010},
+     PAGES = {164--191},
+   MRCLASS = {20D06 (20E28 20G40 68Q25)},
+       DOI = {10.1112/S1461157009000035},
+       URL = {https://doi.org/10.1112/S1461157009000035},
+}
+
+
```

Comment:
Not that important indeed, but since you added this information for all the other bibliography entries as well, it's much nicer this way in terms of consistency :)